### ISSUE TITLE
Issue with remote desktop from a system running at a different DPI level

### ISSUE THREAD

**Issue by AaronKelley** [Open-Shell/Open-Shell-Menu#783]
**Describe the bug**
In at least some cases, when the desktop DPI scaling level changes, Open Shell Menu doesn't "realize" and update the Start button size and position.  I ran into this when connecting to an existing desktop session via Remote Desktop from a system that was running at a different DPI scaling level.

**To Reproduce**
1. I'm logged in at "workstation A" with a 4K display and the DPI scaling level set to 200%.
2. I go sit down to use "workstation B" with a 1080p display and the DPI scaling level set to 100%.
3. I open Remote Desktop Connection on "workstation B" and connect to my running session on "workstation A".
4. ...The Open Shell Menu "Start button" is quite large.

**Expected behavior**
Open Shell Menu should realize that the DPI scaling level has changed and update the icon accordingly.

**Screenshots**
Expected: ![](https://snz04pap001files.storage.live.com/y4m3XMKK_G6CCipmkObD2zo1cpukulsw0wLT5hSQGqnrxboB2BYBtQsVAKfihegWjr0cmO-c03N-Ak0uoNzBAbc7UVU7oD-bRKqaaigHpQoczX07fg1LLg1tc2tVgvr7opTJ2NYOKHbgLH4mjM5jxFCJYSx7S_ZQgqMBkI1vLc2nB9E6udovCfXz_7JwLXX73bk?width=253&height=68&cropmode=none)

Actual: ![](https://snz04pap001files.storage.live.com/y4mak5DcV9tCpVMTGcRnR2T7oN5Ioze8jkNqw026Cb3zsgYZciPXUofVRwspvK0FoDaH-PaXQBwOxEzvvSE0yFPT9JVsNS5zw6tbLpB1mSoGxVh_UGXMyQG3rC16CVxdZhtzW86yhrLqJd7DLCBd6JKG9_mujNuVgNc6tpIMGKoDaKxIVM4AoyqsiuZ_U6FqOMk?width=289&height=63&cropmode=none)

**Version:**
 - Open-Shell: 4.4.160
 - OS: Windows 10 21H1

**Comment by AaronKelley** [comment]
Realized that the "Settings" window also has some layout issues when this happens...

![](https://snz04pap001files.storage.live.com/y4mw9UHHAD9wUi1ML1vjb54OinBsbBbd-RVBB4WLDmY0cMOG1o-x196EFiLZ6fUy_X5u_HwbQYa9sRb06OgiLhULeehkKFEeWc0DDacUU1tacV9IYZtvwe1coiET-0CjXXXXZ5Z9EPOKl69P4mDtgpzscrZ7mba8n9BOX4FJIGwtnnUFTk1rDbznIIGOXN7IttQ?width=676&height=590&cropmode=none)

**Comment by anonimuos** [comment]
The menu has an Override System DPI setting. What is it set to on the remote system? 0 or some custom value? Try right clicking it and changing to "Set as default" ?

**Comment by AaronKelley** [comment]
It's set to 0 which appears to be the default value.

I noticed that it also doesn't update properly if I just go to Settings and change the DPI value locally.  (So the whole Remote Desktop experiment is not needed.)  ...I guess maybe it doesn't support dynamically changing the DPI and I need to close out and restart Open Shell Menu in this case?

**Comment by anonimuos** [comment]
Correct, dynamically changing DPI wasn't added by Ivo. Restarting the menu is required.
